tests/airgapped/katib/README.md

```diff
@@ -0,0 +1,20 @@
+# Testing Katib in airgapped
+
+This directory is dedicated to testing Katib in an airgapped environment.
+
+## Prerequisites
+
+Prepare the airgapped environment and deploy CKF by following the steps in [Airgapped test scripts](https://github.com/canonical/bundle-kubeflow/tree/main/tests/airgapped#testing-airgapped-installation).
+
+Once you run the test scripts, the `kubeflowkatib/simple-pbt:v0.16.0` image used in the `simple-pbt` experiment will be included in your airgapped environment. It's specifically added in the [`get-all-images.sh` script](../../../scripts/airgapped/get-all-images.sh).
+
+## How to test Katib in an Airgapped environment
+1. Connect to the dashboard by visiting the IP of your airgapped VM. To get the IP run:
+```
+lxc ls | grep eth0
```

Comment:
This will work as long as you don't have any other lxc container running.

As an improvement, we could create a [LXD project](https://documentation.ubuntu.com/lxd/en/latest/explanation/projects/), but for now we could tell users which container they should look for.

Reply:
good point, updated